### PR TITLE
common: Unify the pixits setting

### PR DESCRIPTION
**Pull request by mkasenberg**
and add re-setting them up after recovery.

### CODE REVIEW

File: autoptsclient_common.py [hunk 1]
```diff
@@ -1294,6 +1298,37 @@ def run_recovery(args, ptses):
     except:
         traceback.print_exc()
 
+    try:
```
Comment (michal-narajowski) [inline autoptsclient_common.py:1301]: Do we need this try/except here? If there are issues with a particular IXIT then we should handle it individually and let other IXIT values be set. Ideally, we would not cause exceptions in PTS, but just return an error from autopts server.

File: autoptsclient-zephyr.py [hunk 2]
```diff
@@ -78,7 +69,7 @@ def cleanup(self):
 def main():
     """Main."""
 
-    client = ZephyrClient(get_iut, "zephyr_")
+    client = ZephyrClient(get_iut, "zephyr")
```
Comment (michal-narajowski) [inline autoptsclient-zephyr.py:72]: ZephyrClient constructor should pass "zephyr" parameter. Same with Mynewt and Bluez.

File: autoptsclient-zephyr.py [hunk 3]
```diff
@@ -53,15 +53,6 @@ def check_args(self, args):
...
-        autoprojects.gap.set_pixits(ptses[0])
-        autoprojects.dis.set_pixits(ptses[0])
-        autoprojects.sm.set_pixits(ptses[0])
-        autoprojects.l2cap.set_pixits(ptses[0])
-        autoprojects.gatt.set_pixits(ptses)
-        autoprojects.mesh.set_pixits(ptses)
-        autoprojects.mmdl.set_pixits(ptses)
-
     def setup_test_cases(self, ptses):
         self.test_cases = autoprojects.gap.test_cases(ptses[0])
```
Comment (michal-narajowski) [inline autoptsclient-zephyr.py:57]: You could always pass all PTS instances to `profile.test_cases()`
Also, I think ideally you wouldn't need this setup_test_cases in here but in a common client.

Comment (michal-narajowski) [reply autoptsclient-zephyr.py:57]: But maybe that could be done in a next PR.

Comment (michal-narajowski) [reply autoptsclient-zephyr.py:57]: Also see if we can remove `kernel_image` argument.

**Review by michal-narajowski — CHANGES_REQUESTED** [review]

File: autoptsclient_common.py (hunk 1 above)
Comment (mkasenberg) [reply autoptsclient_common.py]: No, we don't actually need it :) After reinit_pts above is called, setup_project_pixits(ptses) should work fine.

File: autoptsclient-zephyr.py (hunk 3 above)
Comment (mkasenberg) [reply autoptsclient-zephyr.py:60]: Sure, I'll see what we can do :+1:

**Review by michal-narajowski — APPROVED** [review]
LGTM

**Pull request merged**